app/assets/stylesheets/_colors.scss

```diff
@@ -0,0 +1,29 @@
+$smoke: #08090a;
+$smoke-90: #202428;
+$smoke-80: #363d44;
+$smoke-70: #4d5760;
+$smoke-60: #64707d;
+$smoke-50: #7d8a97;
+$smoke-40: #99a3ad;
+$smoke-30: #b5bdc4;
+$smoke-20: #d2d6db;
+$smoke-10: #eef0f1;
+$smoke-00: #f9fafa;
+
+$snowflake: #fff;
+
+$dolphin: #3b49df;
+$dolphin-darker: #1a2be5;
+$dolphin-lighter: #8d95f2;
+
+$avocado: #26d9ca;
+$avocado-darker: #1ab3a6;
+$avocado-lighter: #79ece2;
+
+$peanut-butter: #ffcf4c;
+$peanut-butter-darker: #f5b400;
+$peanut-butter-lighter: #ffe499;
+
+$roses-are-red: #dc1818;
```

Comment:
for simplicity's sake, would naming this color `$rose` vs `$roses-are-red` be easier? it's three words and 2 dashes!

Reply:
Yeap, good idea. I'll rename it but also want to keep it unique:P How about `$rose-wine` haha? :DD

If we wanna stick to one-words we could also replace `$peanut-butter` with just `$peanut` or `$mustard` :DDD

Reply:
I've changed it to `$raspberry` so our accent colors are food-related... Because why not :D And I've ended up with `$peanut` as well.

Hmm.. Dolphin is not a food... Unless you eat dolphins..  Maybe lets go with `$plum` ?

Reply:
oooooo

I like this food theme :) `$plum` is fun.